### ISSUE TITLE
1.3.2. bootloader not recognized

### ISSUE THREAD

**Issue by mroz22** [trezor/trezor-suite#2264]
Device:

```
Features (30 bytes) {
    bootloader_mode: True,
    firmware_present: False,
    major_version: 1,
    minor_version: 3,
    patch_version: 2,
    vendor: 'bitcointrezor.com',
}

```

1. open suite, wait for load
1. connect device
1. only one device-change event is emitted and nothing happens.
https://github.com/trezor/connect/blob/f427ab4a0b9a7de782846c5f0cdbed79802c9c24/src/js/device/Device.js#L237-L245
sending initialize before getFeatures helps

**Comment by prusnak** [comment]
The thing is bootloader before 1.4.0 did not react to GetFeatures because the message was introduced when bootloader 1.4.0 was released.

Sending Initialize helps, because it responds with the Features message.

**Comment by prusnak** [comment]
I suggest to use GetFeatures and only if there is no response for several seconds to asks for features via Initialize.

**Comment by slowbackspace** [comment]
fixed in trezor-connect 8.1.13
